### PR TITLE
Rename Example.Repo to Friends.Repo for clarity

### PR DESCRIPTION
**Pull request by nelsonic**
+ [x] fixes issue described in: https://github.com/elixirschool/elixirschool/issues/1908

### CODE REVIEW

**Comment by tajchumber** [comment]
@nelsonic thanks for this PR, what you've done here is correct but
I'm not sure why the app is called `friends` but we generate a repo module called `Example.Repo`.
Shouldn't the repo called `Friends.Repo`?

What I suggest is change the app name to `example` or replace all references of `Example.Repo` with `Friends.Repo`.

What do you think @elixirschool/developers ?

**Comment by nelsonic** [comment]
@tajchumber agree that would be a much better fix. 👍 
(_I was just trying to minimise the changes made. 
but if others agree, happy to create the bigger PR._) 📝

**Comment by nelsonic** [comment]
@tajchumber thanks for the insightful review/comment. 🥇 
I've made the update you suggested: `Example.Repo` >> `Friends.Repo` 📝 
I followed through the example in `iex` to confirm everything is working:
image

Should be good to merge once other @elixirschool/developers agree. 
Thanks.

**Comment by doomspork** [comment]
This is awesome! Thank you @nelsonic !

File: en/lessons/ecto/basics.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-version: 2.1.1
+version: 2.1.2
```
Comment (doomspork) [inline en/lessons/ecto/basics.md]: @tajchumber should we bump the patch or the minor here?

Comment (tajchumber) [reply en/lessons/ecto/basics.md]: I think it's a good idea :)

File: en/lessons/ecto/basics.md [hunk 2]
```diff
@@ -57,18 +57,18 @@ All communication to the database will be done using this repository.
...
 This is the configuration file for our `Example` application
 
 ```elixir
-config :friends, Example.Repo,
+config :friends, Friends.Repo,
   adapter: Ecto.Adapters.Postgres,
   database: "friends_repo",
   username: "postgres",
-  password: "",
+  password: "postgres",
```
Comment (tajchumber) [inline en/lessons/ecto/basics.md]: Maybe here we should specify here that postgres credentials may differ?

File: en/lessons/ecto/basics.md (hunk 1 above)
Comment (doomspork) [reply en/lessons/ecto/basics.md]: @nelsonic could we make this `2.2.0`? That will allow translators to see they need to apply to change (which you can feel free to do in a follow-up PR!)

Comment (nelsonic) [reply en/lessons/ecto/basics.md]: @doomspork makes sense. updating now. 👍

File: en/lessons/ecto/basics.md (hunk 2 above)
Comment (nelsonic) [reply en/lessons/ecto/basics.md]: @tajchumber this is the default password on most installations. but happy to remove it. 👍

**Comment by nelsonic** [comment]
@tajchumber & @doomspork I've made the updates as suggested. 
Thanks for reviewing. 👍

**Comment by doomspork** [comment]
Thank you @nelsonic!

**Pull request merged**